server/routes/testutils/testData.ts

```diff
@@ -1,24 +1,14 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { PrisonerGender } from '../../data/DraftAdjudicationResult'
 import PrisonerSimpleResult from '../../data/prisonerSimpleResult'
 import { Location } from '../../data/PrisonLocationResult'
 import { ReportedAdjudicationStatus } from '../../data/ReportedAdjudicationResult'
 
-export type OtherData = {
-  displayName: string
-  friendlyName: string
-  issuingOfficer: string
-  prisonerLocation: string
-  formattedDateTimeOfDiscovery: string
-  dateTimeOfDiscovery: string
-  dateTimeOfIssue: string
-  formattedDateTimeOfIssue: string
-}
-
 export default class TestData {
   completedAdjudication = (
     adjudicationNumber: number,
     prisonerNumber: string,
-    otherData?: OtherData,
+    otherData?: any,
```

Comment:
Using `any` here because my plan is to be able to use this to insert various different types of data into the function in order to give the correct adjudication data for whatever test we need it for